pkg/distributor/distributor_test.go

```diff
@@ -1777,32 +1777,6 @@ func BenchmarkDistributor_Push(b *testing.B) {
 			},
 			expectedErr: "labels size bytes exceeded",
 		},
-		"timestamp too old": {
```

Comment:
Should we add these tests back since we added the config back?

Reply:
Good catch. I forgot add this one